packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -622,14 +615,16 @@ class _CheckboxState extends State<Checkbox> with TickerProviderStateMixin, Togg
           ..previousValue = _previousValue
           ..shape = widget.shape ?? checkboxTheme.shape ?? defaults.shape!
           ..activeSide = activeSide
-          ..inactiveSide = inactiveSide,
+          ..inactiveSide = inactiveSide
+          ..designSpec = designSpec,
       ),
     );
   }
 }
 
 const double _kEdgeSize = Checkbox.width;
 const double _kStrokeWidth = 2.0;
+const double _kCupertinoStrokeWidth = 2.5;
```

Comment:
Is this value copied from CupertinoCheckbox? If you can, add a comment that explains where it came from, like "Eyeballed on a simulator running iOS 17.0."

Reply:
Copied from CupertinoCheckbox: https://github.com/flutter/flutter/blob/54e66469a933b60ddf175f858f82eaeb97e48c8d/packages/flutter/lib/src/cupertino/checkbox.dart#L324